master/getting-started/bare-metal/bare-metal.md

```diff
@@ -127,22 +127,30 @@ is in state "wait-for-ready". The default location for the log file is
 
 ## Initialising the etcd database
 
-If you are not using the container-based installation (see above), Calico
-doesn't (yet) have a tool to initialise the database for
-bare-metal only deployments. To initialise the database manually, make
-sure the `etcdctl` tool (which ships with etcd) is available, then
-execute the following command on one of your etcd hosts:
+If you are using the container-based installation (using the calico/node
+container image), the database is initialized as soon as you start the first
+node instance.
```

Comment:
The calico/node image still initializes everything ... but if you are not using that container then creating a node resource will do that for you,